src/main/kotlin/com/vk/admstorm/ssh/SshConnectionService.kt

```diff
@@ -144,10 +145,12 @@ class SshConnectionService(private var myProject: Project) : Disposable {
 
                 override fun run(indicator: ProgressIndicator) {
                     try {
-                        mySftpChannel = myConnectionBuilder!!.openSftpChannel(2)
-                        mySftpClient = MySshUtils.getSftpClient(mySftpChannel!!)
+                        SshHandler.handle {
+                            mySftpChannel = myConnectionBuilder!!.openSftpChannel(2)
+                            mySftpClient = MySshUtils.getSftpClient(mySftpChannel!!)
 
-                        onSuccessful?.run()
+                            onSuccessful?.run()
+                        }
                     } catch (e: SshException) {
```

Comment:
You have unwrapped the exception in `SshHandler.unwrap`. Now the exception data will not be caught

Reply:
Fixed. Maybe later we’ll do a more detailed processing of each unwound exception if necessary, but for now it’s ok i think